test_sets/SDL5_0/vehicle_data.txt

```diff
@@ -34,3 +34,5 @@
 ./test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_2nd_request_IGNORED.lua
 ./test_scripts/API/VehicleData/UnsubscribeVehicleData/007_UnsubscribeVD_for_2_apps.lua
 ./test_scripts/API/VehicleData/UnsubscribeVehicleData/008_UnsubscribeVD_HMI_responds_with_not_success_codes.lua
+./test_scripts/API/VehicleData/StabilityControlsStatus/001_GetVehicleData_with_one_param_in_stabilityControlsStatus_structure.lua
```

Comment:
The new test scripts should not be added to this test set since SDL 5.0 doesn't support the new vehicle data features.

Reply:
@ShobhitAd Please find update in c4f6f67